extensions/cornerstone/src/customizations/segmentationPanelCustomization.tsx

```diff
@@ -42,66 +42,13 @@ export default function getSegmentationPanelCustomization({ commandsManager, ser
       center: 'Center',
     },
     'segmentationToolbox.config': () => {
```

Comment:
if they all same return, why we have different key name for this then?

Reply:
can't we just have one key and toolbox look that? I assume inside the Configs it is finding the correct representation type right?

Reply:
@sedghi 

The Toolbox knows nothing about the representation type.

We need three toolboxes because we have different tools in each.

Reply:
alright